alfresco-transform-imagemagick/alfresco-transform-imagemagick/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>alfresco-transform-imagemagick</artifactId>
+    <name>Alfresco Transform ImageMagick</name>
+    <packaging>jar</packaging>
+
+    <parent>
+        <artifactId>alfresco-transform-core</artifactId>
+        <groupId>org.alfresco</groupId>
+        <version>2.1.2-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.alfresco</groupId>
+            <artifactId>alfresco-transformer-base</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <!--<dependency>-->
+            <!--<groupId>org.dom4j</groupId>-->
+            <!--<artifactId>dom4j</artifactId>-->
+        <!--</dependency>-->
```

Comment:
Trivial: outside the scope of this JIRA but I wonder if we should just take the opportunity to clean-up and delete this commented out config ?